README.rst

```diff
@@ -351,35 +351,43 @@ It's also possible to skip specific rules by passing the rule ID. For example, t
 
     $ ansible-lint -x ANSIBLE0011 playbook.yml
 
-False Positives: Muting Ansible Lint Warnings
----------------------------------------------
+False Positives: Skipping Rules
+-------------------------------
 
 Some rules are a bit of a rule of thumb. Advanced *git*, *yum* or *apt* usage, for example, is typically difficult to achieve through the modules. In this case, you should mark the task so that warnings aren't produced.
 
-There are two mechanisms for this - one works with all tasks, the other works with the command checking modules.
+To skip a specific rule for a specific task, inside your ansible yaml add ``# noqa [rule_id]`` at the end of the line. If the rule is task-based (most are), add at the end of any line in the task. You can skip multiple rules via a space-separated list.
 
-Use the ``warn`` parameter with the *command* or *shell* module.
-
-Use ``skip_ansible_lint`` tag with any task that should be skipped.
```

Comment:
All documentation for skip_ansible_lint is being removed? To my understanding, the feature is still there, no?

Reply:
It is not being removed (but may be a good candidate for deprecation in the future). I will update the docs to say it is a less-preferred method to skip.